models/schemas.yml

```diff
@@ -58,8 +62,12 @@ models:
       description: Boolean value for whether this model was executed with a --full-refresh flag.
     - name: model_materialization
       description: The configured materialization of the model.
-    - name: execution_time
-      description: The duration of time in seconds for the model to run.
+    - name: compile_started_at
+      description: Timestamp of when a model starts to be compiled.
+    - name: sql_completed_at
```

Comment:
Would `query_completed_at` be a better name? It's not super clear whether `sql_completed_at` refers to the completed compilation or the actual execution of the sql

Reply:
I like it!